etc/default/default_config.yaml

```diff
@@ -80,13 +80,13 @@ reranker:
     name: huggingface
     url: null
     api_key: null
-    model_name: </PATH/TO/INSTALLDIR/RERANKER_MODEL>
+    model_name: </PATH/TO/INSTALLDIR/RERANKER_MODEL> (e.g. BAAI/bge-reranker-v2-gemma)
     device: cuda
     load_in_4bit: false
     load_in_8bit: false
     max_new_tokens: 150
     trust_remote_code: false
-    device_map: null
+    device_map: auto
```

Comment:
Do you recommend setting this to `auto` by default? If so let me know so I can update the ansible template as well